src/js/stores/MarathonStore.js

```diff
@@ -217,10 +236,17 @@ var MarathonStore = Store.createStore({
       case ActionTypes.REQUEST_MARATHON_GROUPS_SUCCESS:
         MarathonStore.processMarathonGroups(action.data);
         break;
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_SUCCESS:
+        MarathonStore.processMarathonDeployments(action.data);
+        break;
       case ActionTypes.REQUEST_MARATHON_GROUPS_ERROR:
         MarathonStore.processMarathonGroupsError();
         break;
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_ERROR:
+        MarathonStore.processMarathonDeploymentsError();
+        break;
       case ActionTypes.REQUEST_MARATHON_GROUPS_ONGOING:
+      case ActionTypes.REQUEST_MARATHON_DEPLOYMENTS_ONGOING:
```

Comment:
For what it's worth, this pattern exists all over the UI for handling ongoing requests. I would vote we decide what to do with them and then take care of them all at once.